storage-proofs-core/src/pieces.rs

```diff
@@ -153,13 +154,16 @@ mod tests {
     fn test_generate_piece_commitment_bytes_from_source() -> Result<()> {
         let some_bytes: Vec<u8> = vec![0; 64];
         let mut some_bytes_slice: &[u8] = &some_bytes;
-        generate_piece_commitment_bytes_from_source::<PoseidonHasher>(&mut some_bytes_slice, 64)
-            .expect("threshold for sufficient bytes is 32");
+        generate_piece_commitment_bytes_from_source::<PoseidonHasher<Fr>>(
```

Comment:
Note: only test using `Fr` because we've already tested Poseidon arity-2 Merkle trees.